guides/mentorship/mentors/README.md

```diff
@@ -0,0 +1,40 @@
+##Mentors
+Thank you for signing up to be a Made Tech mentor. Here’s a little guide for how to prepare and go about the sessions.
+
+The point of doing this is to provide an opportunity for people to expand on their coding skills from professionals in the industry, but also to provide those looking to enter the industry with an insight into what it can be like. 
+
+As a mentor, you will be supporting people at various stages on their tech journeys, these could be people who applied to the Academy, or people who are members or various tech communities or networks. You get to help someone by sharing both your technical and non-technical knowledge and experience, helping them grow on their fundamentals or understand some tricky concepts.
+
+###Mentor / Mentee Pairing
+There are two ways that we will be pairing mentors with mentees:
+####Academy Support
```

Comment:
(Style opinion, feel free to disregard)
There should be an empty line before a heading in markdown, imo.